apps/dashboard/src/components/header-navigation/customer-support-button.tsx

```diff
@@ -0,0 +1,12 @@
+import { RiCustomerService2Line } from 'react-icons/ri';
+import { useBootIntercom } from '@/hooks';
+
+export const CustomerSupportButton = () => {
+  useBootIntercom();
+
+  return (
+    <button id="intercom-launcher">
```

Comment:
Intercom is using this id to initialize it and use as the toggle button for the Intercom bubble